content/post/094-gossipsub-v1.1.md

```diff
@@ -0,0 +1,79 @@
+
+3) [Opportunistic Grafting](https://github.com/libp2p/specs/blob/master/pubsub/gossipsub/gossipsub-v1.1.md#opportunistic-grafting): A new mechanism used to cherry pick well-behaving peers that are currently not part of a node’s mesh. Those well-behaving peers are then opportunistically grafted to improve the mesh’s health in terms of median score of participating peers. 
+
+4) [Prune Peer eXchange](https://github.com/libp2p/specs/blob/master/pubsub/gossipsub/gossipsub-v1.1.md#prune-backoff-and-peer-exchange): with Prune Peer exchange a node receives a set of recommended peers to connect when it gets dropped from some other peer’s mesh. With this, we’ve reduced the dependency on ambient peer discovery mechanisms.
+
+5) [Extended Message Validators](https://github.com/libp2p/specs/blob/master/pubsub/gossipsub/gossipsub-v1.1.md#extended-validators): You can now add your custom message validator so that you can instrument Gossipsub to recognize (accept/reject) which messages are valid/invalid within the context of your application.
+
+The blend of these security measures, together with the properties of the “gossiping mesh” check the two most important boxes for message propagation protocols in permissionless networks, namely: i) fast message propagation, ii) security against attacks that attempt to degrade the quality of the service provided by the network.
+
+All of these upgrades can be consulted in the [Gossipsub v1.1 specification](https://github.com/libp2p/specs/blob/master/pubsub/gossipsub/README.md), which if you ask us, it is a joy to read!
+
+Additionally, with the new release, the [reference implementation of Gossipsub](https://github.com/libp2p/go-libp2p-pubsub) got additional documentation and test coverage.
+ 
+## 🏎 Give Gossipsub v1.1 a test ride
+
+### Chat application example
+
+We’ve prepared a brand new example for you to give the latest Gossipsub a try. In this example, you get to experiment with a Chat application powered by libp2p. It will enable you to join different channels by subscribing to PubSub topics and seeing other peers joining the room as they subscribe to the topic.
+
+<p align="center">
+  <img width="60%" src="https://raw.githubusercontent.com/libp2p/go-libp2p-examples/master/pubsub/chat/chat-example.gif">
+</p>
+
+You can find the code for this example at https://github.com/libp2p/go-libp2p-examples/tree/master/pubsub/chat
+
+### Mockchain example
+
+TODO write this
+
+### Test Gossipsub’s performance with Testground
+
+We are also releasing a brand new libp2p Test Plan that uses Testground to benchmark Gossipsub v1.1.
+
+<p align="center">
+  <img width="60%" src="/header_images/094-gossipsub-v1.1-headerimage.jpeg">
+</p>
+
+TODO INSERT IMAGE HERE of the JUPYTER NOTEBOOK
```

Comment:
@yusefnapora do you happen to have a nice screenshot (gift would be 1000x better) of what looks like to run the baseline test plan ? https://github.com/libp2p/test-plans/tree/master/pubsub